scripts/prepare-msm.sh

```diff
@@ -31,11 +31,12 @@ chmod +x ${DIR}/../msm/msm
 
 # Determine which user is running this script
 setup_user=$USER
+setup_group=`id -gn $USER`
```

Comment:
I know at the moment mycroft scripts use some of both, but I think we should transition to `$()` command substitution to make the code more consistent. Let me know what you think.

Reply:
Reading up on this I can't disagree. Seems like the back-ticks are depreciated even. Will fix

Reply:
The PR has been updated